ocamlformat-help.txt

```diff
@@ -201,6 +201,12 @@ OPTIONS (CODE FORMATTING STYLE)
            `=`). loose does. tight-decl is tight for declarations and loose
            for instantiations. The default value is tight.
 
+       --function-blank-line={compact|preserve-one}
```

Comment:
Maybe "function" is not the right word, it apply to any expressions. I think "sequence" is great. A sequence in the AST is a `a ; b` expression but I think, from a high level perspective, the word can be also used for `let .. in ..`.

Reply:
I reverted to keep the old name.